website/content/en/preview/AWS/provisioning.md

```diff
@@ -358,3 +358,26 @@ Refer to general [Kubernetes GPU](https://kubernetes.io/docs/tasks/manage-gpus/s
 * `amd.com/gpu`: [AMD GPU device plugin for Kubernetes](https://github.com/RadeonOpenCompute/k8s-device-plugin)
 * `aws.amazon.com/neuron`: [Kubernetes environment setup for Neuron](https://github.com/aws/aws-neuron-sdk/blob/master/neuron-deploy/tutorials/tutorial-k8s.rst)
 {{% /alert %}}
+
+### Pod ENI (Security Groups for Pods)
```

Comment:
Could we also show how this can be specified in the `resources` section of the  `podSpec` similar to how we do with `nvidia` above

```yaml
spec:
  template:
    spec:
      containers:
      - resources:
          requests:
            vpc.amazonaws.com/pod-eni: "1"
```

Reply:
Sure thing.

Reply:
>Still, since the below section is describing the process of how to enable this in your cluster, we should phrase these as steps. i.e.
> Enable Security Groups for Pods

I see what you mean, though there are additional steps required to enable Security Groups for pods.  Pod-eni is just a prerequisite.  Perhaps we should just show how to enable pod-eni resources in Karpenter and then refer to the [Security Groups for Pods deployment guide]((https://docs.aws.amazon.com/eks/latest/userguide/security-groups-for-pods.html#security-groups-pods-deployment)) for all the remaining steps.

Something like:
"If you intend to use Security Groups for Pods with Karpenter, refer to the Security Groups for Pods deployment guide for additional steps".

A little background.  The reason I added instructions for how to enable pod-eni in the CNI plugin is to prevent an issue where Pods get stuck in a "Pending" state.  In short, if we enable pod-eni in Karpenter, we must enable it in the VPC CNI plugin as well.

Reply:
I think prefacing this as these are the steps to enable pod-eni resources in Karpenter is reasonable

Reply:
Updated.  Take another look now.